src/lib/pipeline.ml

```diff
@@ -164,13 +168,20 @@ module Full (Bfx: Extended_edsl.Semantics) = struct
   open Option
   module Stdlib = Biokepi.EDSL.Library.Make(Bfx)
 
-  let to_bam ~parameters ~reference_build input =
-    let list_of_inputs = Stdlib.bwa_mem_opt_inputs input in
-    List.map list_of_inputs ~f:(Bfx.bwa_mem_opt ~reference_build ?configuration:None)
+
+  let to_bam ~parameters ~reference_build samples =
+    let sample_to_bam sample =
+      let list_of_inputs = Stdlib.bwa_mem_opt_inputs sample in
+      List.map list_of_inputs ~f:(Bfx.bwa_mem_opt ~reference_build ?configuration:None)
+      |> Bfx.list
```

Comment:
This is an artifact of the TTFI EDSL; this lifts a `list` in OCaml-land into a list in TTFI-land.